src/components/ui/FiltersPart.tsx

```
@@ -0,0 +1,194 @@
+
+const categories = ["Phones", "Tablets", "Laptops", "PC Parts"];
+const conditions = ["New", "Like New", "Good", "Fair", "Poor"];
+const brands = ["Apple", "Samsung", "Google", "Dell", "HP", "Lenovo"];
+const phoneModels = ["iPhone 15", "iPhone 14", "Samsung S24", "Pixel 8"];
+const laptopModels = ["MacBook Pro", "MacBook Air", "Dell XPS", "HP Spectre"];
+const tabletModels = ["iPad Pro", "iPad Air", "Samsung Tab"];
+const storageOptions = ["64GB", "128GB", "256GB", "512GB", "1TB", "2TB"];
+const sellerTypes = ["Spoke with before", "Verified sellers"];
+
+const Badge = ({ label, onRemove }: { label: string; onRemove: () => void }) => (
+  <span className="inline-flex items-center gap-1 px-2 py-1 bg-blue-100 text-blue-800 text-xs font-medium rounded-full cursor-pointer hover:bg-blue-200" onClick={onRemove}>
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)